### PR TITLE
Feat: Create readme for event service

### PR DESCRIPTION
**Pull request by ZixMai**
1.  Created readme for event service

### CODE REVIEW

**Review by Nhsdkk — APPROVED** [review]
LGTM

File: event_service/README.md [hunk 1]
```diff
@@ -0,0 +1,85 @@
+___
+# Event service
+[![Build event service](https://github.com/UnspokenTeam/Calendar/actions/workflows/build_event_service.yaml/badge.svg)](https://github.com/UnspokenTeam/Calendar/actions/workflows/build_event_service.yaml)
+___
+**Установка и описание**
+___
+1. **Образец .env файла**
+	```
+	DATABASE_URL="DATABASE_URL"
```
Comment (Nhsdkk) [inline event_service/README.md:10]: Add comments about variables

File: event_service/README.md [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+       - Установка осуществляется через **docker**, для установки пакета микросервиса введите и выполните команду:
+		```
+    	docker pull ghcr.io/unspokenteam/event_service:latest
+ 		```
+       - Создайте .env файл, укажите его путь в команде и выполните её:
+		```
+		docker run -d -p 8081=8081 —env-file $PATH_TO_ENV ghcr.io/unspokenteam/event_service:latest
+ 		```
+
+	2. **Через poetry**
```
Comment (Nhsdkk) [inline event_service/README.md]: Add proto and prisma client generate steps

**Review by Nhsdkk — CHANGES_REQUESTED** [review]
Make changes

File: event_service/README.md [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+___
+**Установка и описание**
+___
+1. **Образец .env файла**
+	```
+	DATABASE_URL="DATABASE_URL"
+	ENVIRONMENT="ENVIRONMENT"
+	OPENROUTER_API_KEY="API_KEY"
+	```
+	- где:
```
Comment (Nhsdkk) [inline event_service/README.md]: Move this to comments inside dotenv example

File: event_service/README.md [hunk 4]
```diff
@@ -0,0 +1,97 @@
...
+      - DATABASE_URL - адрес базы данных
+      - ENVIRONMENT - вид окружения
+      - OPENROUTER_API_KEY - ключ для взаимодействия с LLM через API от OpenRouter
+___
+2. **Установка**:
```
Comment (Nhsdkk) [inline event_service/README.md:17]: Add pre-installation step to create db

**Review by Nhsdkk — CHANGES_REQUESTED** [review]
Make changes

**Review by MaxPlays35 — APPROVED** [review]

**Review by Nhsdkk — APPROVED** [review]
LGTM

**Pull request merged**